xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -14,11 +14,49 @@
 
 #ifdef __cplusplus
 
+#include <map>
+
 namespace kodi
 {
 namespace addon
 {
 
+class CInstanceInputStream;
+
+class ATTRIBUTE_HIDDEN InputstreamProperty
+  : public CStructHdl<InputstreamProperty, INPUTSTREAM_PROPERTY>
+{
+  friend class CInstanceInputStream;
+
+public:
+  std::string GetURL() const { return m_cStructure->m_strURL; }
+
+  unsigned int GetPropertiesAmount() const
+  {
+    return m_cStructure->m_nCountInfoValues;
+  }
+
+  const std::map<std::string, std::string> GetProperties() const
+  {
+    std::map<std::string, std::string> props;
+    for (unsigned int i = 0; i < m_cStructure->m_nCountInfoValues; ++i)
+      props.insert(std::make_pair<std::string, std::string>(
+          m_cStructure->m_ListItemProperties[i].m_strKey,
+          m_cStructure->m_ListItemProperties[i].m_strValue));
```

Comment:
Minor comment, I would say that the for loop is a single command, but multiple lines, and would qualify for `{}`. I think the additional vertical spacing would also aid readability.